sphinx_needs/css/blank/styles.css

```diff
@@ -90,3 +90,6 @@ table.needs_style_blue_bar {
     background: linear-gradient(to right, transparent 90%, #0fbcf9 10%);
 }
 
+.is_visible{
```

Comment:
This is quite a generic class selector that might conflict with custom end-user CSS.
Can you make it more specific to a need/table/header/... or change the name to something like 'collape_is_visible'?
Also the name is inverted somehow.

Reply:
Changes implemented in the new commit.